core/backend/src/CodeService.ts

```diff
@@ -162,6 +162,8 @@ export namespace CodeService {
   export interface InternalWriteMethods extends WriteMethods {
     /**  @internal */
     reserveFontId(props: CodeService.FontIndexProps): Promise<FontId>;
+    /** @internal */
+    reserveEmbeddedFaceDataId(facesKey: string): Promise<number>;
```

Comment:
@williamkbentley new CodeService method for 5.0.

Reply:
sorry, but i have no idea what you're referring to

Reply:
The line we are commenting on here adds a new `reserveEmbeddedFaceDataId` method to the `CodeService` interface that will need to be implemented in the implementation in code-service repo. I thought you had taken over maintenance of that package, but it appears not. I will submit a PR.